tests/kafkatest/version.py

```diff
@@ -252,3 +252,7 @@ def get_version(node=None):
 V_3_5_0 = KafkaVersion("3.5.0")
 V_3_5_1 = KafkaVersion("3.5.1")
 LATEST_3_5 = V_3_5_1
+
+# 3.6.x versions
+V_3_6_0 = KafkaVersion("3.6.0")
+LATEST_3_6 = V_3_6_0
```

Comment:
I had noticed that 3.6 had been removed when I was working on this change and just thought I'd add it back in this PR since it seemed the convention was to add the new version whenever we created a release branch for the prior version. Other than some OCD to follow the existing pattern, I'm indifferent to keeping it or leaving it out since as you said, it's not in use currently

Reply:
Ok, we'll just leave it here and add it back in now. Thanks.